courses/machine_learning/deepdive2/time_series_prediction/labs/2_feature_engineering.ipynb

```diff
@@ -0,0 +1,1298 @@
+{
```

Comment:
<p>Should this be a heading or bolded?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>